src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -1304,9 +1304,9 @@ TEST_CASE_VERSIONS("txenvelope", "[tx][envelope]")
                             tx1 = b.tx({setOptions(
                                 setMasterWeight(1) | setLowThreshold(1) |
                                 setMedThreshold(2) | setHighThreshold(3))});
-                            tx2 = b.tx(
-                                {payment(root, 100), root.op(payment(b, 100))},
-                                b.getLastSequenceNumber() + 1);
+                            tx2 = root.tx(
+                                {b.op(payment(root, 100)), payment(b, 100)},
+                                root.getLastSequenceNumber() + 2);
```

Comment:
@sisuresh I'm re-reviewing this test, and actually I'm not sure why `root.getLastSequenceNumber() + 2` works here. Shouldn't it work without specifying seqnum at all, i.e. the right seqnum will be assigned automatically? I get an exception without the "root.getLastSequenceNumber() +2".

Reply:
Something I noticed recently is that once the seqNum is set in `TestAccount` from the loaded account, it needs to be incremented in `TestAccount` using `nextSequenceNumber`. I'm wondering if `root.create` isn't incrementing the seqNum, resulting in this off by one issue.

Reply:
ah, there's ` root.setOptions(setSigner(sk1));` on line 1313 which consumes root.getLastSequenceNumber() + 1 first, this is why we need `root.getLastSequenceNumber() + 2`. I confirmed that root.create properly consumes a seqnum as well (since it calls `nextSequenceNumber`). So I believe this change is fine then.

Reply:
Ah yeah good catch